src/pudl/transform/ferc1.py

```diff
@@ -2660,6 +2749,7 @@ def transform(
         "plants_pumped_storage_ferc1": PlantsPumpedStorageFerc1TableTransformer,
         "purchased_power_ferc1": PurchasedPowerFerc1TableTransformer,
         "electric_energy_account_sources_ferc1": ElectricEnergyAccountSourcesFerc1TableTransformer,
+        "electric_energy_account_dispositions_ferc1": ElectricEnergyAccountDispositionsFerc1TableTransformer,
```

Comment:
What do you think about swapping in `electric_energy_sources_ferc1` and `electric_energy_dispositions_ferc1` for all of the references to these tables?

Reply:
yea this is much nicer